packages/expo-updates/android/src/main/java/expo/modules/updates/loader/FileDownloader.java

```diff
@@ -312,6 +310,19 @@ private static Request setHeadersForUrl(Uri url, UpdatesConfiguration configurat
       );
     }
 
+    if (extraHeaders != null) {
+      Iterator<String> keySet = extraHeaders.keys();
+      while (keySet.hasNext()) {
+        String key = keySet.next();
+        requestBuilder.header(key, extraHeaders.optString(key, ""));
+      }
+    }
+
+    requestBuilder = requestBuilder
+      .header("Expo-Platform", "android")
+      .header("Expo-Api-Version", "1")
```

Comment:
```suggestion
      .header("Expo-API-Version", "1")
```

(Unrelated change but standard HTTP headers like X-XSS-Protection use uppercase abbreviations. This won't cause any issues since our server lowercases all headers because of HTTP/2 and also headers have been case-insensitive for at least 22 years: https://www.w3.org/Protocols/rfc2616/rfc2616.html)